pkg/flink/lsp/lsp_completer_ws.go

```diff
@@ -80,7 +80,7 @@ func (w *WebsocketLSPClient) refreshWebsocketConnection() {
 	}
 }
 
-func NewLSPClientWS(getAuthToken func() string, baseUrl, organizationId, environmentId string) LspInterface {
+func NewClientWs(getAuthToken func() string, baseUrl, organizationId, environmentId string) LspInterface {
```

Comment:
How about `NewWebsocketClient()` for people who are unfamiliar with what "WS" means?

Reply:
that's better 👍